draft-irtf-cfrg-opaque.md

```diff
@@ -1972,13 +1873,104 @@ This document makes no IANA requests.
 
 # Acknowledgments
 
-The OPAQUE protocol and its analysis is joint work of the author with Stas
+The OPAQUE protocol and its analysis is joint work of the author with Stanislaw
 Jarecki and Jiayu Xu. We are indebted to the OPAQUE reviewers during CFRG's
 aPAKE selection process, particularly Julia Hesse and Bjorn Tackmann.
 This draft has benefited from comments by multiple people. Special thanks
 to Richard Barnes, Dan Brown, Eric Crockett, Paul Grubbs, Fredrik Kuivinen,
 Payman Mohassel, Jason Resch, Greg Rubin, and Nick Sullivan.
 
+# Alternate Key Recovery Mechanisms {#alternate-key-recovery}
+
+Client authentication material can be recovered in different ways. In fact, the original
+definition of OPAQUE proposed to derive an encryption key from the OPRF output, use it
+to encrypt a generated or imported private key, and include the result in the envelope.
+
+In the following we show how a future specification can define that method. Other
+methods MAY introduce alternate `InnerEnvelope` contents.
+
+## External Mode {#external-mode}
```

Comment:
I would drop all of this text entirely, and just say that it's possible to implement another form of Envelope with a different protocol version.

Reply:
I thought we said to move it to the appendix